GoogleForCreators/web-stories-wp
# Animation: Effect Chooser Component

## Summary

- Adds the Animation Effect Chooser to Storybook with Actions

![Kapture 2020-11-06 at 08 53 24](https://user-images.githubusercontent.com/1738349/98379989-97e59000-200d-11eb-94a7-75460753273e.gif)


## Relevant Technical Choices

- Uses CSS keyframes for performant animations over JS
- Adds aria labels with direction or rotation for each button so it's spoken by a screen reader

## To-do

- None

## User-facing changes

- None; all in storybook

## Testing Instructions

1. Visit Storybook
2. View Animations > Effect Chooser
3. See the animations loop *when the animation button is hovered*

---

#3804
